### ISSUE TITLE
[Via AMP AC] AMP Runtime

### ISSUE THREAD

**Issue by jorydotcom** [ampproject/meta-tsc#35]
At the AMP AC meeting on Monday ([see notes][1]), some AC members reported issues with size of the AMP runtime. Is this a known issue? If so, where is it tracked?

[1]: https://github.com/ampproject/meta-ac/pull/79/commits/1f68844ac31ebf6bb421a0163fe6e55983f4aeaf

(👋 quick intro: I'm Jory and I'll be helping @tobie with AMP AC facilitation this year. excited to meet you all at some point!)

**Comment by rudygalfi** [comment]
Hi there @jorydotcom! @nainar may have pointers to where existing discussions are tracked or can loop in additional folks.

**Comment by nainar** [comment]
Hi @jorydotcom! @ampproject/wg-infra and @ampproject/wg-runtime can help here. My understanding is we have a check that runs on every PR to make sure that the runtime size isn't being increased unless approved by someone from the Approvers' WG. 

Is there a specific concern? The notes seem to indicate it's a general concern but I could be misreading this.

**Comment by choumx** [comment]
Some more color on the specific issues would be helpful. I didn't see much in the meeting notes.

**Comment by tobie** [comment]
Thanks for your patience, folks. @pdelgadorodriguez just published a [writeup](https://docs.google.com/document/d/1_enrn4JFrs3L-UEII2J8x7En03asq_5w8yMTtRNODrE/edit?usp=sharing) for https://github.com/ampproject/meta-ac/issues/80 which is also relevant for this issue.

**Comment by kristoferbaxter** [comment]
Are the documents in question running the amp optimizer?

**Comment by rsimha** [comment]
> My understanding is we have a check that runs on every PR to make sure that the runtime size isn't being increased unless approved by someone from the Approvers' WG.

Correct. Today, it's possible for the owners of any of `amphtml`'s JS output files to set a size increase threshold and get notified when a PR pushes the file past that threshold. More information can be found [here](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/bundle-size/README.md) and [here](https://github.com/ampproject/amp-github-apps/tree/master/bundle-size).

**Comment by rudygalfi** [comment]
Have issues mentioned here been documented elsewhere such that this could be closed out?

**Comment by tobie** [comment]
@rudygalfi Could you move it to the AC's repository instead?
